docs/monitoring/metrics.md

```diff
@@ -426,7 +426,7 @@ class MyMapper extends RichMapFunction[Long,Long] {
 
 Every metric is assigned an identifier and a set of key-value pairs under which the metric will be reported.
 
-THe identifier is based on 3 components: the user-defined name when registering the metric, an optional user-defined scope and a system-provided scope.
+The identifier is based on 3 components: the user-defined name when registering the metric, an optional user-defined scope and a system-provided scope.
```

Comment:
@elbaulp I think use "a user-defined name" to replace "the user-defined name" would be better, what's your opinion? @tillrohrmann

Reply:
I think you are right @yanghua. Will change it will merging this PR.